ibm_catalog.json

```diff
@@ -65,18 +66,32 @@
               "role_crns": [
                 "crn:v1:bluemix:public:iam::::role:Administrator"
               ],
-              "service_name": "iam-identity"
+              "service_name": "All Account Management services",
+              "notes": "Administrator access is required for all Account Management services for resource group."
+            },
+            {
+              "role_crns": [
+                "crn:v1:bluemix:public:iam::::role:Administrator"
+              ],
+              "service_name": "All Identity and Access enabled services",
+              "notes": "(Optional) Administrator access is required for enable provisioning of some CBRs only. It is not required if provision_cbr is set to false"
             }
           ],
           "architecture": {
+             "features": [
+              {
+                "title": " ",
+                "description": "Configured to use IBM secure by default standards, but can be edited to fit your use case."
```

Comment:
This description is not applicable for this DA since there is no "Full-configurable" / "Securoty-enforced" variations. In here I would mention a custom description of what this variation does